tests/test_install_requires.py

```
@@ -38,47 +38,48 @@
+class ParseRequirementsTestSuite(BasicUsageTestSuite):
 
-	__module__ = """tests.test_install_requires"""
+	__module__ = "tests.test_install_requires"
 
 	requirements_file = None
 	"""stores the temporary requirements file path for testing"""
 
-	def setUp(self):
-		super(TestParseRequirements, self).setUp()
+	def setUp(self) -> None:
+		super(ParseRequirementsTestSuite, self).setUp()
```

Ruff (0.8.2)

Use `super()` instead of `super(__class__, self)`

Remove `__super__` parameters

(UP008)